drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-runtime/README.md

```diff
@@ -23,14 +23,12 @@ Some minimal tests may be done in the following way:
 VSCODE Integration
 ------------------
 
-1. clone https://github.com/kiegroup/kogito-tooling (Temporary: Use branch yesamer/KOGITO-777)
-2. (Temporary) create WAR of runtime webapp
-3. (Temporary) copy **content** of the war inside kogito-tooling/packages/kie-bc-editors-unpacked/scesim
-4. Launch yarn run init on kogito-tooling directory
-5. Launch yarn run build:prod (yarn run build:fast to skip tests)
-6. Launch VSCode (you can find it in a path similar to: kogito-tooling/packages/vscode-extension-pack-kogito-kie-editors/.vscode-test/vscode-1.43.0/VSCode-linux-x64/bin/code )
-7. In VSCode, open kogito-tooling/packages/vscode-extension-pack-kogito-kie-editors/ folder
-8. Launch DEBUG MODE (F5)
-9. Try to open a SCESIM file or to create a new one
+1. clone https://github.com/kiegroup/kogito-tooling 
+2. Launch yarn run init on kogito-tooling directory
+3. Launch yarn run build:prod (yarn run build:fast to skip tests)
+4. Launch VSCode (you can find it in a path similar to: kogito-tooling/packages/vscode-extension-pack-kogito-kie-editors/.vscode-test/vscode-1.43.0/VSCode-linux-x64/bin/code )
```

Comment:
Shouldn't we mention here to launch it with code --enable-proposed-api kiegroup.vscode-extension-pack-kogito-kie-editors flag?

Reply:
@dupliaka added!